src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -453,6 +465,27 @@ const ContentListTable = () => {
                           {name}
                           <br />
```

Comment:
oh hrmm a `<br />` 

We only want to do this within spans/paragraph elements, I suppose I must have broken my own roles, oh dear.  

I good method to fix this issue and the spacing between each element here would be to make this Td us flex (or wrap the contents as you've done below). And then use the "gap" css feature to ensure equal distance in all children. 
https://css-tricks.com/almanac/properties/g/gap/

Reply:
removed the br and made the changes you suggested, let me know if that's what you meant. the items in the Td seem to be spaced equally now